### PR TITLE
add TestAli and PageTestAli

### PR DESCRIPTION
**Pull request by eltess**

### CODE REVIEW

File: src/test/java/PageAliExpres.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        super (driver);
+    }
+
+    public WebElement serchFild(){
+        return driver.findElement (By.xpath ("//input[@id='search-form__input']"));
+    }
+    public WebElement closeSpam(){
+        return driver.findElement (By.xpath ("//div[@id='cookies-banner__container__close-btn']"));
+    }
+
```
Comment (StepaniukAndii) [inline src/test/java/PageAliExpres.java:17]: empty line

Comment (StepaniukAndii) [reply src/test/java/PageAliExpres.java:17]: @eltess

File: src/test/java/TestAliExpres.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import org.testng.annotations.Test;
+
+public class TestAliExpres extends TestInit{
+
+    @Test
+
```
Comment (StepaniukAndii) [inline src/test/java/TestAliExpres.java:6]: empty line @eltess

File: src/test/java/TestAliExpres.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    public void startTest(){
+         PageAliExpres pageAliExpres =new PageAliExpres (driver);
+         driver.get ("https://allo.ua/ru/kompjutery/");
+         pageAliExpres.serchFild().click ();
+         pageAliExpres.serchFild().sendKeys ("холодильник");
+    }
+
```
Comment (StepaniukAndii) [inline src/test/java/TestAliExpres.java:13]: empty line @eltess

**Review by StepaniukAndii — CHANGES_REQUESTED** [review]

File: src/test/java/PageAliExpres.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PageAliExpres extends BasePage {
+
+    public PageAliExpres(WebDriver driver) {
+        super (driver);
+    }
+
+    public WebElement serchFild() {
```
Comment (Znamm9) [inline src/test/java/PageAliExpres.java:11]: All field that returns elements should start from get
```suggestion
    public WebElement getSerchFild() {
```

File: src/test/java/PageAliExpres.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+        return findElement ("//input[@id='search-form__input']");
```
Comment (Znamm9) [inline src/test/java/PageAliExpres.java:12]: remove spaces before method parenthesis
```suggestion
        return findElement("//input[@id='search-form__input']");
```

File: src/test/java/TestAliExpres.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import org.testng.annotations.Test;
+
+public class TestAliExpres extends TestInit {
+
+    @Test
+
+    public void startTest() {
```
Comment (Znamm9) [inline src/test/java/TestAliExpres.java:7]: remove parenthesis and pun name that displays what actually you do in the test
```suggestion
    public void searchForrefrigerators(){
```

File: src/test/java/PageAliExpres.java [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PageAliExpres extends BasePage {
```
Comment (Znamm9) [inline src/test/java/PageAliExpres.java:5]: which exactly page? AliExpres is a site but not a page. Use a clearer name, please
```suggestion
public class AliHomePage extends BasePage{
```

File: src/test/java/TestAliExpres.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+        PageAliExpres pageAliExpres = new PageAliExpres (driver);
+        openUrl ("https://allo.ua/ru/kompjutery/");
+        pageAliExpres.serchFild ().click ();
+        pageAliExpres.serchFild ().sendKeys ("холодильник");
```
Comment (Znamm9) [inline src/test/java/TestAliExpres.java:11]: remove spaces
```suggestion
        PageAliExpres pageAliExpres = new PageAliExpres(driver);
        openUrl("https://allo.ua/ru/kompjutery/");
        pageAliExpres.serchFild().click();
        pageAliExpres.serchFild().sendKeys("холодильник");
```

File: src/test/java/PageAliExpres.java [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+    public PageAliExpres(WebDriver driver) {
```
Comment (Znamm9) [inline src/test/java/PageAliExpres.java:7]: remove spaces
```suggestion
    public PageAliExpres(WebDriver driver){
```

**Review by Znamm9 — CHANGES_REQUESTED** [review]

**Comment by eltess** [comment]
Addd new folder

**Comment by eltess** [comment]
дщщ

File: src/test/java/AliExpres/PageAliExpres.java [hunk 10]
```diff
@@ -0,0 +1,16 @@
+package AliExpres;
+
```
Comment (Johasan) [inline src/test/java/AliExpres/PageAliExpres.java:2]: Delete space

File: src/test/java/AliExpres/TestAliExpres.java [hunk 11]
```diff
@@ -0,0 +1,16 @@
+package AliExpres;
+
```
Comment (Johasan) [inline src/test/java/AliExpres/TestAliExpres.java:2]: Delete space

Comment (eltess) [reply src/test/java/AliExpres/TestAliExpres.java]: Ok man

**Review by eltess — COMMENTED** [review]
check my new  folder

**Review by Johasan — APPROVED** [review]

**Review by StepaniukAndii — APPROVED** [review]

**Pull request closed without merging**